src/@next/Radio/Radio.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useMemo } from 'react';
+import { Label, LabelWrapper, RadioInput, RadioWrapper } from './RadioStyle';
+import { Typography } from '../Typography';
+import { v4 as uuidv4 } from 'uuid';
+import { Colors } from '..';
+
+export type RadioProps = React.HTMLAttributes<HTMLInputElement> & {
+  label?: string;
+  helperText?: string;
+  disabled?: boolean;
+  name?: string;
+  checked?: boolean;
+};
+
+export const Radio = ({ ...props }: RadioProps) => {
+  const uuid = useMemo(() => uuidv4(), []);
```

Comment:
any reason for `useMemo` here?

Reply:
It won't generate a new uuid for each render

Reply:
I can use a `useState` and `useEffect` but using `useMemo` is a shorthand to it

Reply:
But why we even need to use them? I think this is just to assign a unique id right?

Reply:
I still don't think it needs useMemo or any hooks tho, unless it's a heavy task. Using useMemo also requires additional checks, and since we don't care about autogenerated id, it also has no value to preserve it 🤔

Reply:
I have a personal experience where the generated ID is different between the server and client (browser) when using Next, and it produced unexpected bugs; I am afraid it will happen in the future. But if you think it's unnecessary, I will remove it. But still, in my opinion, there is no disadvantages for using the `useMemo.`

Reply:
Fixed